fixer_suites/three_to_two/fixes/fix_reload.py

```diff
@@ -0,0 +1,30 @@
+"""Fixer for reload().
+
+importlib.reload(s) -> reload(s)"""
+
+# Local imports
+from crosswind import fixer_base
+from crosswind.fixer_util import BlankLine, ImportAndCall, Leaf, Name, Node, syms, token, touch_import
+
+
+class FixReload(fixer_base.BaseFix):
+    BM_compatible = True
+    order = "pre"
+
+    PATTERN = """
+    power< 'importlib' trailer< '.' 'reload' > args=any* > |
+    import_name< 'import' 'importlib' >
+    """
+
+    def transform(self, node, results):
+        arg_list = results.get("args")
+
+        if node.type == syms.import_name:
+            new = BlankLine()
+            return new
+
+        elif arg_list:
+            args = arg_list[0]
+            args = args.clone()
+            prefix = node.prefix
+            return Node(syms.power, [Leaf(token.NAME, "reload"), args], prefix=prefix)
```

Comment:
Instead of `Leaf(token.Name` this should probably use the imported `Name` from `fixer_util`.

Reply:
Good suggestion, changed